pkg/config/features/features.go

```diff
@@ -0,0 +1,155 @@
+// Package features enables a way to encode enabled enabled features in a
```

Comment:
disable disbable the enabled enabled typo 

```suggestion
// Package features enables a way to encode enabled features in a
```

Reply:
Typo in my comment to fix my typo not intentional :)